CodeMaid/UI/Dialogs/Options/OptionsViewModel.cs

```diff
@@ -217,10 +217,10 @@ private void OnExportCommandExecuted(object parameter)
             var activeSettingsName = ActiveSettingsName;
             var dialog = new Microsoft.Win32.SaveFileDialog
             {
-                Title = "CodeMaid: Export " + activeSettingsName,
+                Title = StringResourceKey.OptionsViewModel_OnExportCommandExecuted_CodeMaidExport + activeSettingsName,
                 FileName = "CodeMaid",
                 DefaultExt = ".config",
-                Filter = "Config files (*.config)|*.config|All Files (*.*)|*.*"
+                Filter = StringResourceKey.OptionsViewModel_OnImportCommandExecuted_ConfigFilesConfigConfigAllFiles
```

Comment:
This auto-generated name is very awkward.. perhaps ConfigFileFilter would be clearer.

Reply:
Yes, it's very long, I submit a refactoring code?

Reply:
That would be great, thanks.

Reply:
e279ed8